### PR TITLE
refactor landing page

### PR DESCRIPTION
**Pull request by guiadco**
- Migrate style

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-- package-ecosystem: bundler
+- package-ecosystem: npm
   directory: "/"
   schedule:
     interval: daily
```
Comment (Xide) [inline .github/dependabot.yml:6]: Might want to schedule monthly instead, as new npm dependencies are much more frequent.

File: .github/workflows/deploy-gh.yaml [hunk 2]
```diff
@@ -1,41 +1,48 @@
----
```
Comment (Xide) [inline .github/workflows/deploy-gh.yaml:1]: Why is this file commented ?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+.PHONY+=install
+install:
+	npm ci
+
+.PHONY+=build
+build: install
+	npm run build
+
+local: build
```
Comment (Xide) [inline Makefile:11]: should depend on install instead of build.

File: src/App.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="leading-normal tracking-normal text-white gradient" style="font-family: 'Source Sans Pro', sans-serif;">
+    <nav-bar />
+    <hero />
+    <header-waves />
+    <main-content />
+    <pricing />
+    <faq />
+    <integrations />
+    <equipe />
```
Comment (Xide) [inline src/App.vue:10]: should be named `team`

File: src/App.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    <footer-waves />
+    <temoignage />
```
Comment (Xide) [inline src/App.vue:12]: should be `testimonies`

File: src/App.vue [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    PageFooter,
+  },
+
+ 
+})
+</script>
+
+<style>
+.gradient {
+  /* background: linear-gradient(90deg, #ADB6FF 0%, #D4E09B 100%); */
```
Comment (Xide) [inline src/App.vue:54]: you can remove comments here

File: src/components/CardsSection.vue [hunk 7]
```diff
@@ -0,0 +1,89 @@
+<template>
```
Comment (Xide) [inline src/components/CardsSection.vue:1]: File not used in the project

File: src/components/Hero.vue [hunk 8]
```diff
@@ -0,0 +1,35 @@
+<template>
+    <div class="pt-24">
+      <div class="container px-3 mx-auto flex flex-wrap flex-col md:flex-row items-center">
+        <!--Left Col-->
+        <div class="flex flex-col w-full md:w-2/5 justify-center items-start text-center md:text-left">
+          <p class="uppercase tracking-loose w-full">Transcript as a service</p>
+          <h1 class="my-4 text-5xl font-bold leading-tight">
```
Comment (Xide) [inline src/components/Hero.vue:7]: Text still overflow, maybe try with a smallest font

File: src/components/Hero.vue [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            Votre scribe numérique
+          </h1>
+          <p class="leading-normal text-2xl mb-8">
+             Transcrivez ce que vous voulez, où vous voulez.
+          </p>
+          <!-- <button class="mx-auto lg:mx-0 hover:underline bg-white text-gray-800 font-bold rounded-full my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
+            Subscribe
+          </button> -->
```
Comment (Xide) [inline src/components/Hero.vue:15]: You can remove this comment

File: src/components/Hero.vue [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        </div>
+        <!--Right Col-->
+        <!-- <div class="flex flex-col md:w-1/5" /> -->
+        <div class="flex flex-col w-full md:w-3/5 py-6 text-center" style="height: 512px">
+          <svg-icon name="illustration" class=" w-full md:w-5/5 z-10 h-full" />
+          <!-- <img class="w-full md:w-4/5 z-50" src="hero.png" /> -->
```
Comment (Xide) [inline src/components/Hero.vue:21]: You can remove this comment

File: src/components/MainContent.vue [hunk 11]
```diff
@@ -0,0 +1,60 @@
+<template>
+    <section id="features" class="bg-white border-b py-8">
+      <div class="container max-w-5xl mx-auto m-8">
+        <h1 class="w-full my-2 text-5xl font-bold leading-tight text-center text-gray-800">          
+          Fonctionnalité
```
Comment (Xide) [inline src/components/MainContent.vue:5]: missing trailing `s`

File: src/components/PageFooter.vue [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+            <a class="text-gray-800 no-underline hover:no-underline font-bold text-xl lg:text-2xl" href="#">
+              <svg-icon name="logo" class="h-8 fill-current inline" viewBox="0 0 512.005 512.005" />
+              
+              <!-- <svg class="h-8 fill-current inline" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512.005 512.005">
+                <rect fill="#2a2a31" x="16.539" y="425.626" width="479.767" height="50.502" transform="matrix(1,0,0,1,0,0)" />
+                <path
+                  class="plane-take-off"
+                  d=" M 510.7 189.151 C 505.271 168.95 484.565 156.956 464.365 162.385 L 330.156 198.367 L 155.924 35.878 L 107.19 49.008 L 211.729 230.183 L 86.232 263.767 L 36.614 224.754 L 0 234.603 L 45.957 314.27 L 65.274 347.727 L 105.802 336.869 L 240.011 300.886 L 349.726 271.469 L 483.935 235.486 C 504.134 230.057 516.129 209.352 510.7 189.151 Z "
+                />
+              </svg> -->
```
Comment (Xide) [inline src/components/PageFooter.vue:15]: You can remove this comment

File: src/components/PageFooter.vue [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+              2021 - Cursiv
```
Comment (Xide) [inline src/components/PageFooter.vue:16]: 2021 should not be using this class, it is unnecessary large

File: src/components/PageFooter.vue [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+              <a
+                href="https://www.iubenda.com/privacy-policy/42361767"
+                class="iubenda-black iubenda-embed"
+                title="Privacy Policy"
+              >Privacy Policy</a>
+            </a>
+          </div>
+          <!-- <div class="flex-1">
```
Comment (Xide) [inline src/components/PageFooter.vue:24]: all comments below can be deleted

File: src/components/Pricing.vue [hunk 15]
```diff
@@ -0,0 +1,72 @@
+<template>
```
Comment (Xide) [inline src/components/Pricing.vue:1]: All this file can be deleted. We can just explain the quotas and pricing in the FAQ below

File: tailwind.config.js [hunk 16]
```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  purge: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    extend: {
+      colors: {
+        'primary': '#D4E09B',
+        'secondary': '#ADB6FF'
+      }
```
Comment (Xide) [inline tailwind.config.js:9]: This can be deleted, it was for the other site theme

**Review by Xide — COMMENTED** [review]
You can track all image / binary resources using git lfs
https://docs.github.com/en/github/managing-large-files/versioning-large-files/configuring-git-large-file-storage

File: .github/workflows/deploy-gh.yaml (hunk 2 above)
Comment (guiadco) [reply .github/workflows/deploy-gh.yaml]: slove

File: Makefile (hunk 3 above)
Comment (guiadco) [reply Makefile]: done

File: src/App.vue (hunk 4 above)
Comment (guiadco) [reply src/App.vue]: done

File: src/App.vue (hunk 5 above)
Comment (guiadco) [reply src/App.vue]: done

File: src/components/PageFooter.vue (hunk 12 above)
Comment (guiadco) [reply src/components/PageFooter.vue]: done

File: src/components/PageFooter.vue (hunk 14 above)
Comment (guiadco) [reply src/components/PageFooter.vue]: done

File: .github/dependabot.yml (hunk 1 above)
Comment (guiadco) [reply .github/dependabot.yml]: done

File: src/components/Pricing.vue (hunk 15 above)
Comment (guiadco) [reply src/components/Pricing.vue]: done

File: tailwind.config.js (hunk 16 above)
Comment (guiadco) [reply tailwind.config.js]: done

File: src/components/PageFooter.vue (hunk 13 above)
Comment (guiadco) [reply src/components/PageFooter.vue]: done

**Comment by Xide** [comment]
Refactor mostly done @guiadco, you can review the changes

**Review by guiadco — COMMENTED** [review]
LGTM

**Pull request merged**